azure_functions_worker/dispatcher.py

```diff
@@ -145,8 +146,9 @@ async def dispatch_forever(self):
             logging_handler = AsyncLoggingHandler()
             root_logger = logging.getLogger()
 
-            # Don't change this unless you read #780 and #745
-            root_logger.setLevel(logging.INFO)
+            log_level = logging.INFO if not is_envvar_true(
```

Comment:
You can use the is_envvar_false method instead of the not is_envvar_true

Reply:
i originally thought too but it fails my tests because this method returns False if env key is unset.

```
def is_envvar_false(env_key: str) -> bool:
    if os.getenv(env_key) is None:
        return False

    return is_false_like(os.environ[env_key])
```